src/App.tsx

```diff
@@ -1,11 +1,31 @@
 import React from "react";
 import Example from "./components/Example";
 import Layout from "./components/layout/Layout";
+import PollutedLocationCard from "./components/pollutedLocationCard/PollutedLocationCard";
+import PollutedLocationDTO from "./types/backEnd/PollutedLocationDTO";
+
+type PollutedLocation = PollutedLocationDTO & {
+  title?: string;
+};
 
 const App: React.FC = () => {
+  const pollutedLocation: PollutedLocation = {
+    id: "a950ab10-d394-468b-b2ef-776135a65b96",
+    title: "Unknown Title",
+    location: {
+      longitude: 121.151,
+      latitude: 121.151,
+    },
+    radius: 20,
+    severity: "low",
```

Comment:
Nitpicking: In the backend our Enum values start with an upper-case letter. So here "low" -> "Low" for the sake of keeping everything consistent.